src/domain/httpsactuatorserver.py

```diff
@@ -0,0 +1,95 @@
+"""HTTPS Actuator server module"""
```

Comment:
This is a standalone application? If so Please consider it moving it away from the `domain` folder, together with the `actuator.py` file

Reply:
nit: also why using the `http` module? seems very convoluted. Flask/FastAPI make this super easy and way more portable

Reply:
> This is a standalone application? If so Please consider it moving it away from the `domain` folder, together with the `actuator.py` file
Both modules will need to be referenced in other modules by the code that Stefano will write (in the same way as it was done for ftpserver.py and camera.py).

Reply:
> nit: also why using the `http` module? seems very convoluted. Flask/FastAPI make this super easy and way more portable

That could be an idea, but I thought it would be better to avoid importing additional external libraries to keep everything lighter, since the logic to implement is quite simple.

Reply:
many of the library we use (torch, openvino...) have size footprint that is on the tensor to the hundreds of MB. The size that a library like fastAPI brings is very negligible in comparison